### ISSUE TITLE
The Snackbar 'success' variant does not meet color contrast requirements

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#233]
![Screen Shot 2019-08-28 at 3 18 33 PM](https://user-images.githubusercontent.com/3613392/63885888-4b350c80-c9a7-11e9-9d96-06e24c385eb0.png)

The color contrast of white on green has a contrast ratio of 1.97 to 1 rather than 4.5 to 1

The color in question:
```
#8CCA3A
```

Making a green color sufficiently dark to meet standards would be a pretty dramatic departure from the current design. It might be worth re-thinking how the status of a toast is communicated - i.e., with an icon - to get around this problem.

**Comment by jonambas** [comment]
Closing – fixed in hibana
